data_steward/cdr_cleaner/cleaning_rules/missing_concept_record_suppression.py

```diff
@@ -0,0 +1,166 @@
+"""
+Remove records that contain concept_ids that do not belong in the vocabulary.
+
+Original Issues: DC1601
+"""
+
+# Python imports
+import logging
+
+# Third party imports
+from google.cloud.exceptions import GoogleCloudError
+
+# Project imports
+from cdr_cleaner.cleaning_rules.deid.concept_suppression import AbstractBqLookupTableConceptSuppression
+from constants.cdr_cleaner import clean_cdr as cdr_consts
+from common import JINJA_ENV, CDM_TABLES
+from utils import pipeline_logging
+from resources import get_concept_id_fields
+
+LOGGER = logging.getLogger(__name__)
+
+SUPPRESSION_RULE_CONCEPT_TABLE = 'missing_vocabulary_concepts'
+
+CREATE_OR_REPLACE_CLAUSE = JINJA_ENV.from_string("""
+CREATE OR REPLACE TABLE `{{project_id}}.{{sandbox_id}}.{{concept_suppression_lookup_table}}` AS
+{{query}}    
+""")
+
+MISSING_CONCEPTS_QUERY = JINJA_ENV.from_string("""
+    SELECT DISTINCT
+        t.{{concept_id_field}} concept_id
+    FROM `{{project_id}}.{{dataset_id}}.{{tablename}}` t
+    LEFT JOIN `{{project_id}}.{{dataset_id}}.concept` c
+        ON c.concept_id = t.{{concept_id_field}}
+    WHERE c.concept_id IS NULL
+        AND (t.{{concept_id_field}} IS NOT NULL AND t.{{concept_id_field}} <> 0)
+""")
+
+
+class MissingConceptRecordSuppression(AbstractBqLookupTableConceptSuppression):
+
+    def __init__(self, project_id, dataset_id, sandbox_dataset_id):
```

Comment:
`table_namer` object needs to be used and set. 

```suggestion
    def __init__(self, project_id, dataset_id, sandbox_dataset_id, table_namer=table_namer):
```

Also needs to be passed through the `init` method.